aosp_go_e2303.mk

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TARGET_KERNEL_CONFIG := aosp_kanuti_tulip_defconfig
+
+# Inherit from those products. Most specific first.
+$(call inherit-product, device/sony/tulip/device.mk)
+$(call inherit-product, frameworks/native/build/phone-xhdpi-2048-dalvik-heap.mk)
+# Inherit common Android Go configurations
```

Comment:
do not remove
$(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_base_telephony.mk)